libs/base/form/src/renderers/default-form.renderer.ts

```diff
@@ -161,20 +161,22 @@ export class DefaultFormRenderer implements FormRenderer {
     field: FormFieldDefinition,
     value?: string
   ): TemplateResult {
+    const { pattern } = this.fieldValidationPattern(field);
     return html`
       <oryx-input
         .label=${field.label}
         floatLabel=${ifDefined(field.floatLabel)}
-        class=${this.getClassMap(field)}
+        .style=${this.resolveStyles(field)}
       >
         <input
-          .name=${field.id}
-          .value=${value ?? ''}
+          name=${field.id}
+          value=${value ?? ''}
           placeholder=${ifDefined(field.placeholder)}
           type="number"
           min=${ifDefined(field.min)}
           max=${ifDefined(field.max)}
           ?required=${field.required}
+          pattern=${ifDefined(pattern)}
```

Comment:
`title` is also required to get the correct validation message, check the input field's builder